lib/services/local/sessiontracker.go

```diff
@@ -123,7 +123,15 @@ func (s *sessionTracker) GetActiveSessionTrackers(ctx context.Context) ([]types.
 			return nil, trace.Wrap(err)
 		}
 
-		sessions[i] = session
+		if session.Expiry().After(time.Now().UTC()) {
+			sessions[i] = session
+		} else if item.Expires.IsZero() { // Check if the expiry is not set.
```

Comment:
What about expired items? We silently ignore them?

I suggest using a switch so we can add a comment on the "expired, non-zero branch".

Reply:
Sure, added a switch with a comment